examples/platform/linux/AppMain.cpp

```diff
@@ -201,6 +201,7 @@ int ChipLinuxAppInit(int argc, char * const argv[], OptionSet * customOptions)
         PrintOnboardingCodes(LinuxDeviceOptions::GetInstance().payload);
     }
 
+#if 0
```

Comment:
How about making this a command-line argument?

My thinking is: either this is useful code, so we should have it as an argument option, or it is obsolete code, in which case we should delete it rather than effectively commenting it out. `#if 0` comments it out.

Reply:
I personally think it's obsolete because it obviously duplicates calling `PrintOnboardingCodes` function just few lines above. I think we can actually remove this.

Reply:
BTW, the "Custom Commission Flow" is an advanced usage because it would be hooked with the info in DCL. So I don't think this will be commonly used. The codes remaining to be here are just for reference. Actually it can be removed.

Here's few extract from spec `5.1.4. Manual Pairing Code`
```
...  For this case, a commissioner SHOULD use Vendor ID and Product ID to lookup the 
CommissioningCustomFlow field in the Distributed Compliance Ledger (DCL) to determine 
which of these values applies for this Vendor ID and Product ID combination. ...
```

Reply:
Lets delete the code then. It can be resurrected from git history if really needed.

Reply:
Hi Andy, the code is removed as next commit https://github.com/project-chip/connectedhomeip/pull/22409/commits/ab16c74dbcff5ac397f587c61386586fa500595c.
Thanks for the comment.